federatedscope/mf/dataset/movielens.py

```diff
@@ -75,7 +97,16 @@ def __init__(self, root, num_client, train_portion=0.9, download=True):
                                "You can use download=True to download it")
 
         ratings = self._load_meta()
-        self._split_n_clients_rating(ratings, num_client, 1 - train_portion)
+
+        self.processed_data = os.path.join(self.root, self.base_folder,
+                                           'processed_data.pkl')
+        if os.path.exists(self.processed_data):
```

Comment:
I don't think line 104-105 will be executed, since a new exp directory (`self.root`) will be created for each time.